0.5.0 release (#102)
user-cont/release-bot: CHANGELOG.md

# 0.5.0

> * Now you can specify labels in release-conf.yaml file

Reviewer: ```suggestion
* You can specify github labels in release-conf.yaml configuration file
  which should be applied on a pull request created by release bot.
```

> * Documentation update

Reviewer: ```suggestion
* Documentation is updated and is more clear now.
```

> * Fix bug that bot tried to release old releases again

Reviewer: ```suggestion
* The bot no longer tries to release old versions again.
```
